app/mutations/ProjectRevision/updateProjectRevision.ts

```diff
@@ -0,0 +1,53 @@
+import type { Environment } from "react-relay";
+import type {
+  updateProjectRevisionMutationVariables,
+  updateProjectRevisionMutation as updateProjectRevisionMutationType,
+} from "updateProjectRevisionMutation.graphql";
+import BaseMutation from "mutations/BaseMutation";
+import { graphql } from "react-relay";
+
+const mutation = graphql`
+  mutation updateProjectRevisionMutation($input: UpdateProjectRevisionInput!) {
+    updateProjectRevision(input: $input) {
+      projectRevision {
+        id
+        formChangesByProjectRevisionId {
+          edges {
+            node {
+              id
+              newFormData
+              formDataSchemaName
+              formDataTableName
+            }
+          }
+        }
+      }
+    }
+  }
+`;
+
+const updateProjectRevisionMutation = async (
```

Comment:
do we want to export a hook instead?

Reply:
maybe we can just export the mutation, and have the pages use the standard `useMutation` hook when needed